openmls/src/framing/message_in.rs

```diff
@@ -97,24 +97,29 @@ impl MlsMessageIn {
     /// Returns the wire format.
     pub fn wire_format(&self) -> WireFormat {
         match self.body {
-            MlsMessageInBody::PrivateMessage(_) => WireFormat::PrivateMessage,
-            MlsMessageInBody::PublicMessage(_) => WireFormat::PublicMessage,
-            MlsMessageInBody::Welcome(_) => WireFormat::Welcome,
-            MlsMessageInBody::GroupInfo(_) => WireFormat::GroupInfo,
-            MlsMessageInBody::KeyPackage(_) => WireFormat::KeyPackage,
+            MlsMessageBodyIn::PrivateMessage(_) => WireFormat::PrivateMessage,
+            MlsMessageBodyIn::PublicMessage(_) => WireFormat::PublicMessage,
+            MlsMessageBodyIn::Welcome(_) => WireFormat::Welcome,
+            MlsMessageBodyIn::GroupInfo(_) => WireFormat::GroupInfo,
+            MlsMessageBodyIn::KeyPackage(_) => WireFormat::KeyPackage,
         }
     }
 
     /// Extract the content of an [`MlsMessageIn`] after deserialization for use
     /// with the [`MlsGroup`] API.
-    pub fn extract(self) -> MlsMessageInBody {
+    pub fn extract(self) -> MlsMessageBodyIn {
         self.body
     }
 
+    /// Try to convert the message into a [`ProtocolMessage`].
+    pub fn try_into_protocol_message(self) -> Result<ProtocolMessage, ProtocolMessageError> {
+        self.try_into()
+    }
```

Comment:
Do we really need this in addition to the `TryFrom` implementation? I would think some documentation on the `ProtocolMessage` and `MlsMessageIn` structs should be enough.